Expensify/App
# Fix error on iOS Payments page

### Details
Changes:
1. Removes `ScrollView`
2. Passes `ListHeaderComponent` prop to `PaymentMethodList`

### Fixed Issues
$ https://github.com/Expensify/App/issues/7087

### Tests
1. Launch the iOS app
2. Navigate to `Settings > Payments`
3. Verify that no error is thrown and you can still see the account balance.

### QA Steps
Steps above.

### Tested On


### Screenshots


#### Web

https://user-images.githubusercontent.com/22219519/148604724-dcf14af8-f573-479f-918e-b097b6804487.mov

#### Mobile Web

https://user-images.githubusercontent.com/22219519/148604643-85795d5f-723e-455e-a3d7-85a466960957.mp4

#### Desktop

https://user-images.githubusercontent.com/22219519/148605577-34b62bf0-37b1-4030-878e-e08f499ea2f1.mov

#### iOS

https://user-images.githubusercontent.com/22219519/148604517-25de1029-0f05-42f8-94c2-b420ff1d2a30.mp4

#### Android

https://user-images.githubusercontent.com/22219519/148604546-41fee23a-7600-4eea-9109-2026c681835a.mov
